src/compressed_tensors/quantization/quant_args.py

```diff
@@ -24,13 +25,54 @@
 
 __all__ = [
     "FP8_DTYPE",
+    "FP8_E4M3_DATA",
+    "FP4_E2M1_DATA",
     "QuantizationType",
     "QuantizationStrategy",
     "QuantizationArgs",
     "round_to_quantized_type",
     "ActivationOrdering",
 ]
 
+
+@dataclass
+class FloatArgs:
+    exponent: int
+    mantissa: int
+    bits: int
+    max: float
+    min: float
+    dtype: Optional[torch.dtype] = None
+
+
+@dataclass
+class FloatArgsFP4E2M1(FloatArgs):
```

Comment:
you can avoid having both `FloatArgsFP4E2M1` and `FP4_E2M1_DATA` by not using a data class

```python3
class FP4E2M1(FloatArgs):
    exponent=2
    mantissa=1
    bits=4
    max=6.0
    min=-6.0
```

Reply:
Good idea. I'll make the udpates. Thanks!